WordPress/src/main/java/org/wordpress/android/ui/posts/PostUtils.java

```diff
@@ -115,4 +128,146 @@ public static void trackSavePostAnalytics(Post post, SiteModel site) {
                 // No-op
         }
     }
+
+    public static boolean isPublishable(PostModel post) {
+        return !(post.getContent().isEmpty() && post.getExcerpt().isEmpty() && post.getTitle().isEmpty());
```

Comment:
I see this is the same code in Post.java, but that seems weird to me to allow publishing post with an empty content and title but non empty excerpt.